assert 1 != 1, "error"
print("unreachable...")